mne/minimum_norm/tests/test_inverse.py

```diff
@@ -708,6 +709,56 @@ def test_io_inverse_operator():
     _compare(inv_prep, inv_prep_prep)
 
 
+@testing.requires_testing_data
+def test_apply_mne_inverse_cov():
+    """Test MNE with precomputed inverse operator on cov."""
+    raw = read_raw_fif(fname_raw, preload=True)
+    # use 10 sec of data
+    raw.crop(0, 10)
+    raw.filter(1, None)
+    label_lh = read_label(fname_label % 'Aud-lh')
+    inverse_operator = read_inverse_operator(fname_full)
+
+    data_cov = compute_raw_covariance(raw, tmin=raw.times[0],
+                                      tmax=raw.times[-1], tstep=None)
+    with pytest.raises(ValueError, match='has not been prepared'):
+        apply_inverse_cov(data_cov, raw.info, 1, inverse_operator,
+                          lambda2=lambda2, prepared=True)
+    inverse_operator = prepare_inverse_operator(inverse_operator, nave=1,
+                                                lambda2=lambda2, method="dSPM")
+
+    # only test pick_ori="normal" as that has a clear expectation of results
+    pick_ori = "normal"
+    first_test_flag = True
+    for method in ['MNE', 'dSPM', 'sLORETA']:
+        stc_raw = apply_inverse_raw(raw, inverse_operator, lambda2, method,
+                                    label=label_lh, nave=1, pick_ori=pick_ori,
+                                    prepared=True)
```

Comment:
for testing with pick_ori != 'normal' you can call apply_inverse_raw with pick_ori = 'vector' and to the sum over orientations in the test. It should match what your code does.